### ISSUE TITLE
A contentblock with a single unchecked checkbox is not rendered

### ISSUE THREAD

**Issue by blaues0cke** [bolt/bolt#7465]
I am using the following template:

```
{% extends 'partials/_master.twig' %}

{% block main %}
    {% for block in record.contentblocks %}
        {{ dump(block.block) }}
    {% endfor %}
{% endblock main %}
```

To display this `contentblock` of my `contenttypes.yml`:

```
pages:
    name: Seiten
    singular_name: Seite
    fields:
        title:
            type: text
            class: large
            group: content
            label: Titel
        slug:
            type: slug
            uses: title
        contentblocks:
            type: block
            label: Blöcke
            group: content
            fields:
                footer:
                    label: Footer
                    fields:
                        negative_margin_top:
                            type: checkbox
                            label: Negativer Abstand nach Oben anzeigen

        seo:
            type: seo
            group: SEO-Einstellungen
        template:
            type: templateselect
            filter: '*.twig'
            group: content
    taxonomy: [ groups ]
    recordsperpage: 20
    default_status: published
    record_template: page.twig
    listing_template: pages.twig
```
Problem: The block is only rendered if `negative_margin_top` is `true`. If `negative_margin_top` is `false` the block does *not* appear in `record.contentblocks`. A workaround is to add another text field.

**Comment by eduardomart** [comment]
I think that's the nature of checkboxes, if they're not there you can assume the `false` value.

**Comment by blaues0cke** [comment]
@eduardomart Yes, but the checkbox value is only a subfield of the block. There is no reason to skip the block rendering att all only because of the fact that all its fields are `!$var`. I think bug would also occur if there are multiple unchecked child checkboxes.

**Comment by rossriley** [comment]
Fixed hopefully in #7503 
Feel free to reopen if there's still any issues.
